### DISCUSSION TITLE
is there a way to route to signup instead of signIn

### DISCUSSION THREAD

**Discussion by yashwanthkrishnab** [zitadel/zitadel#5703] [Q&A] [answered]
can we route the user to signup page ,forget password page instead of going through signIn default

**Comment by hifabienne** [comment, marked as answer]
Hei @yashwanthkrishnab 
With the prompt `create` you can route directly to the signup page.
You can find everything in the docs: https://zitadel.com/docs/apis/openidoauth/endpoints#additional-parameters
Or try it out directly with our OIDC playground: https://zitadel.com/docs/apis/openidoauth/authrequest

**Reply by yashwanthkrishnab** [reply]
thanks @hifabienne 
can we also route directly to forgot password page

**Reply by thevenukan** [reply]
Hi @hifabienne as you replied to @yashwanthkrishnab it is possible to redirect only for create, login & select_account screens but we do have the requirement to redirect the user to reset_password, forgot password screens as well so do we have any alternative to archive the same or is there any way to generate authRequestID through apis so we can archive it in a different way with that authRequestID please let us know your feedback thx

**Reply by hifabienne** [reply]
hei @thevenukan, no this is currently not possible. But with the new login/register API we are currently working on. you would be able to implement your own ui and do the requests to our api. https://github.com/zitadel/zitadel/issues/5015

**Reply by thevenukan** [reply]
thank you @yashwanthkrishnab
